### PR TITLE
bump version wallet latest

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: scripts/macos_installer.py [hunk 1]
```diff
@@ -14,14 +14,14 @@ def __init__(self):
...
-            self.THYRA_WALLET_PLUGIN_URL = "https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.9/wallet-plugin_darwin-arm64.zip"
+            self.THYRA_WALLET_PLUGIN_URL = "https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.10/wallet-plugin_darwin-arm64.zip"
             self.THYRA_WALLET_BINARY_FILENAME = "wallet-plugin_darwin-arm64"
             self.THYRA_WALLET_ZIP_FILENAME = "wallet-plugin_darwin-arm64.zip"
         elif platform.machine() == "x86_64":
             self.THYRA_SERVER_URL = "https://github.com/massalabs/thyra/releases/latest/download/thyra-server_darwin_amd64"
             self.THYRA_APP_URL = "https://github.com/massalabs/Thyra-Menu-Bar-App/releases/latest/download/ThyraApp_darwin-amd64"
             self.MKCERT_URL = "https://dl.filippo.io/mkcert/latest?for=darwin/amd64"
-            self.THYRA_WALLET_PLUGIN_URL = "https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.9/wallet-plugin_darwin-amd64.zip"
+            self.THYRA_WALLET_PLUGIN_URL = "https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.10/wallet-plugin_darwin-amd64.zip"
```
Comment (ASAPSegfault) [inline scripts/macos_installer.py:24]: This link points to something that does not exists

**Review by ASAPSegfault — CHANGES_REQUESTED** [review]

File: scripts/macos_installer.py (hunk 1 above)
Comment (B-Naoufal) [reply scripts/macos_installer.py:24]: should be latest ? @Thykof  ?

Comment (Thykof) [reply scripts/macos_installer.py:24]: to not break thyra install I would like to specify here a fixed version

Comment (Thykof) [reply scripts/macos_installer.py:24]: @SlnPons latest or 0.0.11?

Comment (SlnPons) [reply scripts/macos_installer.py:24]: Latest please :) 
Based on our [release ready definition](https://www.notion.so/massa-innoteam/Plan-for-the-mainnet-c574da44a4854eb3841a5f2e93a2977c?pvs=4#3d75e410a46f4282aad137485ffb3e3b), all releases should be usable. So the latest is the best to start your MassaStation experience.

**Review by ASAPSegfault — APPROVED** [review]

File: api/test/robot_tests/cmd/keywords.resource [hunk 2]
```diff
@@ -26,7 +26,7 @@ Delete Wallet Plugin
 Install Wallet Plugin
     Log To Console    Installing Massa Wallet Plugin
     ${source}=    Set Variable
-    ...    https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.9/wallet-plugin_${OS}-${ARCH}.zip
+    ...    https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.10/wallet-plugin_${OS}-${ARCH}.zip
```
Comment (peterjah) [inline api/test/robot_tests/cmd/keywords.resource:29]: you don't use latest here ?

Comment (Thykof) [reply api/test/robot_tests/cmd/keywords.resource]: nice catch thanks!

**Review by peterjah — APPROVED** [review]

**Comment by Thykof** [comment]
We had to refactor how we detect that a plugin is already installed: be more precise: check the presence the binary file. That way in unit test we can create the plugin directory in advance, but the innonet wallet file.

**Review by mazmassa — APPROVED** [review]

**Review by thomas-senechal — APPROVED** [review]

**Pull request merged**